### PR TITLE
ADMINAPI-764 | Rate Limit

### PR DESCRIPTION
**Pull request by lchavarriaga**

### CODE REVIEW

File: Application/EdFi.Ods.AdminApi/appsettings.json [hunk 1]
```diff
@@ -33,5 +33,22 @@
...
+        "EnableEndpointRateLimiting": true,
+        "StackBlockedRequests": false,
+        "RealIpHeader": "X-Real-IP",
+        "ClientIdHeader": "X-ClientId",
+        "HttpStatusCode": 429,
+        "IpWhitelist": [],
+        "EndpointWhitelist": [],
+        "GeneralRules": [
+            {
+                "Endpoint": "POST:/Connect/Register",
```
Comment (jagudelo-gap) [inline Application/EdFi.Ods.AdminApi/appsettings.json:48]: we should have the "/connect/token" endpoint as well.

File: Application/EdFi.Ods.AdminApi/appsettings.json [hunk 2]
```diff
@@ -33,5 +33,22 @@
...
+                "Period": "1m",
+                "Limit": 3
```
Comment (jagudelo-gap) [inline Application/EdFi.Ods.AdminApi/appsettings.json:50]: this value might be configurable in appsettings file

**Review by jagudelo-gap — CHANGES_REQUESTED** [review]

File: Application/EdFi.Ods.AdminApi/appsettings.json (hunk 1 above)
Comment (lchavarriaga) [reply Application/EdFi.Ods.AdminApi/appsettings.json:48]: Based on the standup meeting comments, new changes on Rate Limit implementation will be reviewed on Demo meeting with Stephen.

File: Application/EdFi.Ods.AdminApi/appsettings.json (hunk 2 above)
Comment (lchavarriaga) [reply Application/EdFi.Ods.AdminApi/appsettings.json:50]: Based on the standup meeting comments, new changes on Rate Limit implementation will be reviewed on Demo meeting with Stephen.

**Review by simpat-adam — APPROVED** [review]

**Review by jagudelo-gap — APPROVED** [review]

**Pull request merged**